packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.service.js

```diff
@@ -86,4 +86,10 @@ export default class OvhManagerPccDatacenterService {
       },
     );
   }
+
+  postAddNsxtEdge(serviceName, datacenterId) {
```

Comment:
No need to mention http method is the name
```suggestion
  addNsxtEdge(serviceName, datacenterId) {
```